dlt/cli/init_command.py

```diff
@@ -746,3 +778,49 @@ def init_pipeline_at_destination(
 
         copied_files: Dict[str, str] = {dest_path: src_path for src_path, dest_path in copy_files}
         return copied_files, source_type
+
+
+# def inform_about_config_flavor_message(required_secrets: Sequence[WritableConfigValue], section: str) -> None:
+#     # get all unique secret hints and print their config flavor menu's
+#     possible_choices = set(secret.hint for secret in required_secrets if section in secret.sections)
+#     for hint in possible_choices:
+#         print_config_flavor_menu_from_union(hint)
+
+
+def check_for_config_flavors(config_values: Dict[str, WritableConfigValue], section: str) -> str:
```

Comment:
this is hacky code. I thought it would be simpler to find out if a config-hint class has flavors.